lib/v2/nua/radar.js

```diff
@@ -1,7 +1,31 @@
 module.exports = {
     'nua.edu.cn': {
         _name: '南京艺术学院',
-        grad: [
+        index: [
+            {
+                title: '官网信息',
+                docs: 'https://docs.rsshub.app/university.html#nan-jing-yi-shu-xue-yuan',
+                source: ['/:type/list.htm'],
+                target: '/nua/index/:type',
+            },
+        ],
+        sxw: [
+            {
+                title: '双馨网',
+                docs: 'https://docs.rsshub.app/university.html#nan-jing-yi-shu-xue-yuan',
+                source: ['/:type/list.htm'],
+                target: '/nua/sxw/:type',
+            },
+        ],
+        dc: [
+            {
+                title: '设计学院',
+                docs: 'https://docs.rsshub.app/university.html#nan-jing-yi-shu-xue-yuan',
+                source: ['/:type/list.htm'],
+                target: '/nua/dc/:type',
+            },
+        ],
+        gra: [
```

Comment:
```suggestion
        grad: [
```
https://github.com/DIYgod/RSSHub/pull/10528#discussion_r949085674